src/ProjectTemplates/test/Templates.Mvc.Tests/Templates.Mvc.Tests.csproj

```diff
@@ -20,6 +20,14 @@
     <RestoreFolderName>Mvc</RestoreFolderName>
     <TestDependsOnAspNetPackages>true</TestDependsOnAspNetPackages>
     <HelixTimeout>02:30:00</HelixTimeout>
+
+    <!-- These tests fail in Helix in Debian and Mariner due to error -901 -->
+    <!-- Disabling on those OSes until a better fix can be identified -->
+    <SkipHelixQueues>
+      $(HelixQueueDebian12);
+      $(HelixQueueMariner);
+      $(SkipHelixQueues)
+    </SkipHelixQueues>
```

Comment:
Do we need to skip the whole assembly? Or can we just skip the individual failing tests?

Reply:
Nope, this was just an experiment. Thanks for pointing it out though.

Since then I made another PR with a more narrow skip condition which has now been merged: https://github.com/dotnet/aspnetcore/pull/56292

Reply:
Update: no, the above comment is incorrect.

Yes we *do* need to skip the whole assembly since there isn't an individual failing test. It's an unknown timeout problem.

Reply:
> Do we need to skip the whole assembly? Or can we just skip the individual failing tests?

TBH, I think we should probably skip the entire test assembly. I know that is a bit heavy handed but, from looking at a few of the recent test process crashes, there's no correlation between particular tests and expected failures. It's all over the board.